executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -173,6 +173,11 @@ pub fn split_out_machines<'a, T: FieldElement>(
         identities.clone(),
         fixed,
     );
+
+    // Prevent the fixed lookup witnesses to overwrite the base witnesses.
+    let fixed_lookup_witnesses = fixed_lookup.multiplicity_columns();
```

Comment:
At this point, it's irrelevant that it's a multiplicity column, so I think a better name for ths function is `witness_columns`

Reply:
Also, are you sure this works like this? What about the identities that involve the multiplicity columns? Doesn't the machine extractor create a machine for that? Or shouldn't we at least also remove those identities?

Reply:
Ah, maybe they are all filtered out because they involver higher-stage witness columns. Can you check nevertheless?

Reply:
I'm going to check this

Reply:
> Also, are you sure this works like this? What about the identities that involve the multiplicity columns? Doesn't the machine extractor create a machine for that? Or shouldn't we at least also remove those identities?

How can I get them to remove?

Reply:
> Ah, maybe they are all filtered out because they involver higher-stage witness columns. Can you check nevertheless?

Yes, we're filtering here: https://github.com/powdr-labs/powdr/blob/main/executor/src/witgen/mod.rs#L220